lib/ruby_lsp/tapioca/addon.rb

```diff
@@ -62,6 +61,9 @@ def version
 
       sig { params(changes: T::Array[{ uri: String, type: Integer }]).void }
       def workspace_did_change_watched_files(changes)
+        return unless T.must(@global_state).experimental_features
+        return unless @rails_runner_client # Client is not ready
```

Comment:
I personally prefer multiple `unless` calls over combining them and thinking of De Morgan's Laws 😬